src/fastga/models/weight/mass_breakdown/payload.py

```diff
@@ -46,7 +46,24 @@ def setup(self):
         self.add_output("data:weight:aircraft:payload", units="kg")
         self.add_output("data:weight:aircraft:max_payload", units="kg")
 
-        self.declare_partials("*", "*", method="fd")
+        self.declare_partials(
+            "data:weight:aircraft:payload",
+            [
+                "data:TLAR:NPAX_design",
+                "settings:weight:aircraft:payload:design_mass_per_passenger",
+                "data:TLAR:luggage_mass_design",
+            ],
+            method="exact",
+        )
+        self.declare_partials(
+            "data:weight:aircraft:max_payload",
+            [
+                "data:geometry:cabin:seats:passenger:NPAX_max",
+                "settings:weight:aircraft:payload:max_mass_per_passenger",
+                "data:geometry:cabin:luggage:mass_max",
+            ],
+            method="exact",
+        )
```

Comment:
To discuss with you but since some of those partials are constant at one, wouldn't it be more interesting to declare them directly here as constant with value 1 ?